sql/core/src/main/scala/org/apache/spark/sql/parquet/newParquet.scala

```diff
@@ -118,7 +120,7 @@ private[sql] class ParquetRelation2(
     private val maybeDataSchema: Option[StructType],
     private val maybePartitionSpec: Option[PartitionSpec],
     parameters: Map[String, String])(
-    val sqlContext: SQLContext)
+    @transient val sqlContext: SQLContext)
```

Comment:
Why do we want to have `@transient` here? `ParquetRelation2` is not serializable and shouldn't be serialized.